contrib/cf-remote/cf_remote/utils.py

```diff
@@ -74,6 +75,15 @@ def above_package_path():
     return os.path.abspath(path)
 
 
+def is_package_url(string):
+    return bool(re.match("https?://.+/.+\.(rpm|deb|msi)", string))
+
+
+def get_package_name(url):
+    assert(is_package_url(url))
+    return url[url.rfind("/") + 1:]
```

Comment:
I think `rsplit()` would be more straightforward here.

Reply:
Changed.